person.rb

```diff
@@ -0,0 +1,29 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # Getters for @id
+  attr_reader :id
+
+  # Getter and Setter for @name and @age
+  attr_accessor :name
+  attr_accessor	:age
```

Comment:
[OPTIONAL] You can sum it like this : 

`attr_accessor :name, :age `

Reply:
Thanks for the suggestion. Really appreciate it :)
Addressed the comment in latest commit :)